keps/sig-api-machinery/2855-automated-stored-versions/README.md

```diff
@@ -87,24 +87,25 @@ It is desirable for the [kube-storage-version-migrator] to modify a CRD's `statu
 
 ### Goals
 
-- Allow the [kube-storage-version-migrator] to modify a CRD's `status.storedVersions` after a successful migration.
+- Allow the [kube-storage-version-migrator] to safely modify a CRD's `status.storedVersions` after a successful migration.
+- Stop other actors from modifying the storage version while a migration is ongoing.
 
 ## Proposal
 
-In order for the [kube-storage-version-migrator] to safely modify a CRD's `status.storedVersions` field after a successful migration, this KEP proposes that the [kube-storage-version-migrator] introduce a [Validating Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) that prevents changes to a CRD's `spec.versions[*].storage` fields during a migration.
+In order for the [kube-storage-version-migrator] to safely modify a CRD's `status.storedVersions` field after a successful migration, this KEP proposes that the [kube-storage-version-migrator] introduces a [Validating Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) that prevents changes to a CRD's `spec.versions[*].storage` fields during a migration.
 
 The validating admission webhook would need to:
 
-- Intercept all Update and Patch operations against CRDs that include the `ksvm.k8s.io/migrating=true` annotation.
-- Prevent changes to a CRD's `spec.versions[*].storage` fields.
+- Intercept all Update and Patch operations against CRDs.
+- Prevent changes to a CRD's `spec.versions[*].storage` fields if the CRD includes the `ksvm.sigs.k8s.io/migrating=true` annotation.
 
 A summary of the new migration workflow can be found below:
 
 1. A `StorageVersionMigration` CR for the `foo` CRD has been created, possibly by the trigger-controller as [described here](https://github.com/kubernetes-sigs/kube-storage-version-migrator/blob/acdee30ced218b79e39c6a701985e8cd8bd33824/USER_GUIDE.md#storage-version-migrator-in-a-nutshell)). 
-2. The [kube-storage-version-migrator] adds the `ksvm.k8s.io/migrating=true` annotation to the`foo` CRD.
+2. The [kube-storage-version-migrator] adds the `ksvm.sigs.k8s.io/migrating=true` annotation to the`foo` CRD.
 4. The [kube-storage-version-migrator] begins migrating all `foo` resources as it does today.
 5. When a migration is successful, the `foo` CRD's `status.storedVersion` is patched to only include the new stored version.
-6. When a migration is finished, the [kube-storage-version-migrator] removes the `ksvm.k8s.io/migrating=true` annotation from the `foo` CRD.
+6. When a migration is finished, the [kube-storage-version-migrator] removes the `ksvm.sigs.k8s.io/migrating=true` annotation from the `foo` CRD.
```

Comment:
later in the implemention we will also need to add a reconcile logic, i.e. if the migration object is gone the annotation has to be removed too.